### ISSUE TITLE
Activation Token Cannot Be Generated For Both Private Service And Stream Exports With The Same Subject

### ISSUE THREAD

**Issue by glarrick757** [nats-io/nsc#686]
### What version were you using?

nsc version 2.10.0

### What environment was the server running in?

N/A

### Is this defect reproducible?

Create a private service and stream export with the same subject:

```
nsc add export --subject foo --private --account bar
nsc add export --subject foo --private --service --account bar
```

Attempt to make an activation token:
```
nsc generate activation --account bar --target-account <some_other_account_pub_key> --subject foo -o /tmp/activation.jwt
```

All such activation tokens will be of type service. There is no way to make a stream activation token.

`nsc generate activation -i` appears to give the ability to select between the stream and service exports. However, this choice doesn't seem to make a difference - the activation token always has type `service`.

### Given the capability you are leveraging, describe your expectation?

It should be possible to make an activation token for both the private stream export and private service export.

### Given the expectation, what is the defect you are observing?

An activation token can only be created for one of the two types of private exports.

**Comment by aricart** [comment]
correct, you cannot have a service and a stream that have the same subject it must be one or the other.

**Comment by aricart** [comment]
Just to be clear - a stream (is not 'jetstream') - but something that publishes messages that your receive from another account. A service is something you send another account.
